New Resource: `azurerm_network_manager_verifier_workspace_reachability_analysis_intent` (#28956)
hashicorp/terraform-provider-azurerm: website/docs/r/network_manager_verifier_workspace_reachability_analysis_intent.html.markdown

  source_resource_id      = azurerm_linux_virtual_machine.example.id
  destination_resource_id = azurerm_linux_virtual_machine.example.id
  description             = "example"
  ip_traffic {
    source_ips        = ["10.0.2.1"]
    source_ports      = ["80"]
    destination_ips   = ["10.0.2.2"]
    destination_ports = ["*"]
    protocols         = ["Any"]
  }
}
```

## Arguments Reference

The following arguments are supported:

* `name` - (Required) The name which should be used for this Network Manager Verifier Workspace Reachability Analysis Intent. Changing this forces a new Network Manager Verifier Workspace Reachability Analysis Intent to be created.

* `verifier_workspace_id` - (Required) The ID of the Network Manager Verifier Workspace. Changing this forces a new Network Manager Verifier Workspace Reachability Analysis Intent to be created.

* `destination_resource_id` - (Required) The ID of the destination resource. The value can be the ID of either Public internet, Cosmos DB, Storage Account, SQL Server, Virtual machines, or Subnet. Changing this forces a new Network Manager Verifier Workspace Reachability Analysis Intent to be created.

> * `ip_traffic` - (Required) One or more `ip_traffic` blocks as defined below. Changing this forces a new Network Manager Verifier Workspace Reachability Analysis Intent to be created.

Reviewer: Since schema defines this with `maxItems: 1`:

```suggestion
* `ip_traffic` - (Required) An `ip_traffic` block as defined below. Changing this forces a new Network Manager Verifier Workspace Reachability Analysis Intent to be created.
```

Author: fixed
